src/Compilers/CSharp/Test/Emit3/Attributes/AttributeTests_Embedded.cs

```diff
@@ -20,7 +22,8 @@ public void ReferencingEmbeddedAttributesFromTheSameAssemblySucceeds()
             var code = @"
 namespace Microsoft.CodeAnalysis
 {
-    internal class EmbeddedAttribute : System.Attribute { }
+    [Embedded]
```

Comment:
>[Embedded] [](http://example.com/codeflow?start=4&length=10)

It looks like this change (the attribute application) is no longer necessary here and in other similar places.